### PR TITLE
Update scripts to use `increment`

### PR DESCRIPTION
**Pull request by javierhonduco**
Update scripts under `tools/` to use `$map.increment` and documentation examples.

- The 1st commit replaces the increments by 1
- The 2nd commit replace increments of steps bigger than 1, as introduced in #1897 
- The 3rd one updates the documentation

cc/ @bobrik

### CODE REVIEW

File: docs/tutorial_bcc_python_developer.md [hunk 1]
```diff
@@ -564,6 +564,9 @@ int count(struct pt_regs *ctx) {
 	u64 zero = 0, *val;
 
 	bpf_probe_read(&key.c, sizeof(key.c), (void *)PT_REGS_PARM1(ctx));
+
+  // another possibility is using `counts.increment(key);`. It allows a second
```
Comment (palmtenor) [inline docs/tutorial_bcc_python_developer.md]: Indentation seem to be a bit off here and below~

**Comment by palmtenor** [comment]
[buildbot, test this please]

File: docs/tutorial_bcc_python_developer.md (hunk 1 above)
Comment (javierhonduco) [reply docs/tutorial_bcc_python_developer.md]: Changed it to spaces as below

**Comment by yonghong-song** [comment]
LGTM. Thanks for cleanup and using the new interface!

**Pull request merged**